index.d.ts

```diff
@@ -0,0 +1,193 @@
+     * @param {boolean} requireResolversForNonScalar will cause makeExecutableSchema to throw an error if a non-scalar field has no resolver defined. Setting this to `true` can be helpful in catching errors, but defaults to `false` to avoid confusing behavior for those coming from other GraphQL libraries.
+     * @param {boolean} requireResolversForAllFields asserts that _all_ fields have valid resolvers.
+     * @param {boolean} requireResolversForResolveType will require a _resolveType()_ method for Interface and Union types. This can be passed in with the field resolvers as *__resolveType()*. False to disable the warning.
+     * @param {boolean} allowResolversNotInSchema turns off the functionality which throws errors when resolvers are found which are not present in the schema. Defaults to `false`, to help catch common errors.
+     */
+    interface AugmentSchemaResolverValidationOptions {
+        requireResolversForArgs: boolean;
+        requireResolversForNonScalar: boolean;
+        requireResolversForAllFields: boolean;
+        requireResolversForResolveType: boolean;
+        allowResolversNotInSchema: boolean;
+    }
+
+    interface AugmentSchemaDirectives {
+        [key: string]: (next: Promise<any>, src: any, args: RequestArguments, context: any) => Promise<any>;
+    }
+
+    type DirectiveResolvers = Record<string, () => any>;
+
+    /**
+     * AugmentSchemaAuthConfig
+     * @param isAuthenticated   enables `@isAuthenticated` directive, **Optional, defaults to true**
+     * @param hasRole           enables `@hasRole` directive, **Optional, defaults to true**
+     * @param hasScope          enables `@hasScope` directive, **Optional, defaults to true**
+     */
+    interface AugmentSchemaAuthConfig {
+        isAuthenticated?: boolean;
+        hasRole?: boolean;
+        hasScope?: boolean;
+    }
+
+    /**
+     * AugmentSchemaConfig
+     *
+     * @param query     Configure the autogenerated Query fields. Can be enabled/disabled for all types or a list of individual types to exclude can be passed. Commonly used to exclude payload types. **Optional defaults to `true`**
+     * @param mutation  Configure the autogenerated Mutation fields. Can be enabled/disabled for all types or a list of individual types to exclude can be passed. Commonly used to exclude payload types. **Optional, defaults to `true`**
+     * @param debug     Enable/disable logging of generated Cypher queries and parameters. **Optional, defaults to `true`**
+     * @param auth      Used to enable authorization schema directives (@isAuthenticated, @hasRole, @hasScope). If enabled, directives from the graphql-auth-directives are declared and can be used in the schema. If @hasScope is enabled it is automatically added to all generated query and mutation fields. See the authorization guide for more information. **Optional, defaults to `false`**
+     */
+    interface AugmentSchemaConfig {
+        query?: boolean | { exclude: string[] };
```

Comment:
missing `experimental?: boolean;`

Reply:
according to the docs i dont see any `experimental` key https://grandstack.io/docs/neo4j-graphql-js-api#config

Reply:
Also I cant seem to find the usage of any experimental kep diving trhoug the code and into the types of `@graphql-tools` that is being used underneath this function.

Reply:
https://grandstack.io/docs/graphql-schema-generation-augmentation/#experimental-api

Reply:
fixed with: e4a6986